### PR TITLE
VCM-213

### PR DESCRIPTION
**Pull request by mayaalsamarrai**
### Proposed of changes
Describe the technical details of realization provided by you.

### Make sure these boxes are checked:
- [ ] Check all the changes in github PR - files count (non of them are redundant, have meaningful changes, all are added), if target branch is correct
- [ ] Check methods and variable namings - it should be self descriptive, no typos
- [ ] Check you did not introduce breaking changes in API and public models/services.
- [ ] Respect extensibility - https://community.virtocommerce.com/t/extensibility-basics-the-domain-model-and-persistence-layer-extension/141
- [ ] Follow [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself) and [SOLID](https://en.wikipedia.org/wiki/SOLID) principles
- [ ] For unit tests - follow Microsoft best practices: https://docs.microsoft.com/en-us/dotnet/core/testing/unit-testing-best-practices
- [ ] Consolidate solution dependencies in case you are using newer version, update VC module dependencies in module.manifest. Do not upgrade 3rd party packages that are shipped with the platform with the version newer than in the platform.
- [ ] Check code style conventions - https://github.com/VirtoCommerce/styleguide/blob/master/csharp.md
- [ ] Check PR have a concise and descriptive title, follow [git commit message rules](https://github.com/VirtoCommerce/styleguide/blob/master/gitcommits.md)

### CODE REVIEW

**Comment by lnetrebskii** [comment]
Review task https://virtocommerce.atlassian.net/browse/VP-2884 has been created

File: README.md [hunk 1]
```diff
@@ -1 +1,55 @@
...
+1. On 'Edit Payment method' blade you can edit the following parameters:
+
+     1. Activate or de-activate the Payment method using the 'Is Active' button;
+     1. Make partial payments available for clients on the Storefront by switching on the 'Is partial payments available' button;
+1. Once the editing is finished, save the changes made;
+1. All changes made in VC admin will be displayed on the Storefront.
+
+![Edit Payment method](src/docs/media/screen-edit-payment-method.png)
+
+### Edit Settings
```
Comment (yecli) [inline README.md:35]: Need to to mention that it is AuthorizeNET settings, not the general payment method one.
Like 
Edit settings (AuthorizeNET payment method is used as an example)

File: README.md [hunk 2]
```diff
@@ -1 +1,55 @@
...
+
+         1. Authorization/Capture
+         1. Sale
+1. Save the changes if you have edited the payment method settings.
+
+![Settings](src/docs/media/screen-payment-settings.png)
+
+## Documentation 
+
+[Transaction types](https://support.authorize.net/s/article/What-Are-the-Transaction-Types-That-Can-Be-Submitted)
```
Comment (yecli) [inline README.md:53]: Do not think these are required here, true are not related to payment module functionality at all, they are AuthorizeNET method specific.

File: README.md [hunk 3]
```diff
@@ -1 +1,55 @@
...
+
+1. Select the 'Settings' widget on 'Edit payment method' blade;
+1. The following settings can be edited: 
+
+     1. Mode- you can select the option you need from the drop down list;
+     1. Confirmation URL
+     1. Thank you page URL
+     1. Payment action type:
+
+         1. Authorization/Capture
```
Comment (yecli) [inline README.md:45]: Please add to which authorizeNET method each option corresponds:
 Sale - AUTH_CAPTURE
 Authorization/Capture- это AUTH_ONLY

File: README.md [hunk 4]
```diff
@@ -1 +1,55 @@
...
+
+## Overview
+
+The Payment module allows the following options:
+
+1. Register payment methods using the code;
+1. Receive the list of payment methods on UI in VC admin;
+1. Edit payment method settings;
+1. Connect the payment methods to a Store.
+1. The selected payment methods will be available for selection on the Storefront.
```
Comment (yecli) [inline README.md:11]: Need to mention:
API to work with payment method list;

File: README.md (hunk 1 above)
Comment (yecli) [reply README.md:35]: I would remove AuthorizeNET at all and used Manual test payment method tax provider as an example. As it is the one that is shipped with the module. And need to change the screenshosts

File: README.md (hunk 3 above)
Comment (yecli) [reply README.md:45]: In case Manual method would be used - it is not needed. This info should be in the AuthorizeNET docs

**Pull request merged**